k-distribution/tests/regression-new/map-symbolic-tests/failing/lookup-8-spec.k.out

```diff
@@ -0,0 +1,3 @@
+<k>
+  lookupResult ( 2 )
+</k>
```

Comment:
again, i don't understand why it is failed.

i mean, the current output is:
```
lookupResult ( MAP [ Z <- 2 ] [ Z ] )
```
which should have been simplified to `lookupResult ( 2 )`, by the following rule:
```
rule (MAP:Map [ K1 <- V1 ]) [ K2 ] => V1         requires K1  ==K K2
```

do you think you can further look into it?

Reply:
I'm not sure why it's failed either, but we have a record of it now, and can look into it in another PR.